.pre-commit-config.yaml

```diff
@@ -35,7 +35,7 @@ repos:
         files: "\\.sh$"
 
   - repo: https://github.com/ansible/ansible-lint
-    rev: v24.12.2
+    rev: v25.8.1
```

Comment:
v25.1.1 (currently used on master) works on release-2.27 AFAICT (fix the error and does not cause new warnings).